doc/api/http2.md

```diff
@@ -1171,6 +1174,17 @@ server.on('stream', (stream, headers, flags) => {
 });
 ```
 
+#### Event: 'request'
+<!-- YAML
+added: REPLACEME
+-->
+
+* `request` {http2.Http2ServerRequest}
+* `response` {http2.Http2ServerResponse}
+
+Emitted each time there is a request. Note that there may be multiple requests
+per session. See the [Compatibility API](compatiblity-api).
```

Comment:
If we are going to list these here, I think we need to be more explicit about these being part of the compatibility API and calling out the differences between this and the `'stream'` event.